modules/ebi-metagenomics/extractcoords/meta.yml

```diff
@@ -0,0 +1,58 @@
+---
+# yaml-language-server: $schema=https://raw.githubusercontent.com/nf-core/modules/master/modules/meta-schema.json
+name: "extractcoords"
+description: Process output from easel-sfetch to extract SSU and LSU sequences
+keywords:
+  - easel
+  - rrna
+  - ssu
+  - lsu
+tools:
+  - "extractcoords":
+      description: "Process output from easel-sfetch to extract SSU and LSU sequences, uses mgnify-pipelines-toolkit as conda environment"
+      homepage: "https://github.com/EBI-Metagenomics/mgnify-pipelines-toolkit"
+      documentation: "https://github.com/EBI-Metagenomics/mgnify-pipelines-toolkit"
+      tool_dev_url: "https://github.com/EBI-Metagenomics/mgnify-pipelines-toolkit"
+      licence: ["Apache-2.0"]
+
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. `[ id:'sample1', single_end:false ]`
+  - easel_coords_fasta:
+      type: file
+      description: Fasta file output from running esl-sfetch to extract sequences by name
+      pattern: "*.fasta"
+  - matched_seqs_with_coords:
+      type: file
+      description: cmsearchdeoverlap output formatted for use by esl-sfetch
+      pattern: "*.matched_seqs_with_coords"
+
+output:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. `[ id:'sample1', single_end:false ]`
+  - ssu_fasta:
+      type: file
+      description: Fasta file containing the SSU sequences
+      pattern: "*.fasta"
+  - lsu_fasta:
+      type: file
+      description: Fasta file containing the LSU sequences
+      pattern: "*.fasta"
+  - concat_ssu_lsu_coords:
+      type: file
+      description: Concatenated file of coordinates of matches of both SSU and LSU
+      pattern: "*.concat_SSU_LSU_coords"
```

Comment:
Same here. Depending on the type I would rename to {meta}.tsv or something

Reply:
Or worst case scenario {meta}.concat_SSU_LSU_coords.tsv. But I would still go for something shorter ;)

Reply:
same here, it's space separated, but .txt file is probably better